client/my-sites/customer-home/cards/features/reader/index.tsx

```diff
@@ -6,7 +6,7 @@ import iconReaderLightbulb from 'calypso/assets/images/customer-home/reader-ligh
 import withDimensions from 'calypso/lib/with-dimensions';
 import wpcom from 'calypso/lib/wp';
 import { trackScrollPage } from 'calypso/reader/controller-helper';
-import DiscoverNavigation from 'calypso/reader/discover/discover-navigation';
+import DiscoverNavigation from 'calypso/reader/discover/components/navigation/v1';
```

Comment:
This file is from the "Reader in My Home" hackathon experiment we did a couple of years ago. https://github.com/Automattic/dotcom-forge/issues/4546 We can probably remove the whole file, but I've just updated the import statement here for now.